packages/ethereum-contracts/contracts/apps/SuperAppBaseFlow.sol

```diff
@@ -0,0 +1,265 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity >= 0.8.0;
+
+import {ISuperfluid, ISuperToken, ISuperApp, SuperAppDefinitions} from "../interfaces/superfluid/ISuperfluid.sol";
+
+abstract contract SuperAppBaseFlow is ISuperApp {
+    ISuperfluid public _host;
+
+    mapping(ISuperToken => bool) internal _acceptedSuperTokens;
+
+    /// @dev Thrown when the callback caller is not the host.
+    error UnauthorizedHost();
+
+    /**
+     * @dev Initializes the contract by setting the expected Superfluid Host.
+     *      and register which callbacks the Host can engage when appropriate
+     */
+    constructor(ISuperfluid host_, uint256 callBackDefinitions) {
+        _host = host_;
+
+        host_.registerApp(SuperAppDefinitions.APP_LEVEL_FINAL | callBackDefinitions);
+    }
+
+    /**
+     * @dev Expect Super Token involved in callback to be an accepted one 
+     *      This function can be overridden with custom logic and to revert if desired
+     */
+    function _isAcceptedSuperToken(ISuperToken superToken) public view virtual returns (bool) {
```

Comment:
if public, the fn name shouldn't be prefixed with _

Is there a reason why _isAcceptedSuperToken is public while _isAcceptedAgreement is internal?

Reply:
Thanks for the _catch. 

There isn't a reason for _isAcceptedAgreement to be internal but there isn't really much of a reason for it not to be. I can make it public for good measure